### PR TITLE
Refactor page and article workflow to keep history

### PR DESCRIPTION
**Pull request by mutantmonkey**

### CODE REVIEW

File: wuvt/defaults.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 # DJ activity timer for automation/DJ logout in seconds
 DJ_TIMEOUT = 30 * 60
-EXTENDED_DJ_TIMEOUT = 120 * 60
+EXTENDED_DJ_TIMEOUT = 240 * 60
```
Comment (echarlie) [inline wuvt/defaults.py]: any particular reason for this? no DJ should be playing tracks longer than 2 hours reasonably. I played bach's mass in B minor and cheated on the logging, and it was still only about 2 hours of music.

File: migrations/versions/e56b26b6fc2b_article_and_page_revisions.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    sa.Column('content', sa.UnicodeText().with_variant(sa.UnicodeText(length=2**1), 'mysql'), nullable=True),
+    sa.Column('html_summary', sa.UnicodeText().with_variant(sa.UnicodeText(length=2**1), 'mysql'), nullable=True),
+    sa.Column('html_content', sa.UnicodeText().with_variant(sa.UnicodeText(length=2**1), 'mysql'), nullable=True),
+    sa.ForeignKeyConstraint(['article_id'], ['article.id'], ),
+    sa.ForeignKeyConstraint(['author_id'], ['user.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+
+def downgrade():
```
Comment (echarlie) [inline migrations/versions/e56b26b6fc2b_article_and_page_revisions.py]: is there much of a benefit to supporting downgrades?

File: wuvt/defaults.py (hunk 1 above)
Comment (mutantmonkey) [inline wuvt/defaults.py]: Some DJs have had problems with this when doing live shows. I can move this to a separate PR if you prefer.

File: migrations/versions/e56b26b6fc2b_article_and_page_revisions.py (hunk 2 above)
Comment (mutantmonkey) [inline migrations/versions/e56b26b6fc2b_article_and_page_revisions.py]: Doesn't hurt. Also, I think with Alembic you're supposed.

File: wuvt/defaults.py (hunk 1 above)
Comment (ehobgood) [inline wuvt/defaults.py]: Most DJs only have a live performance going on for one hour, two hours max. I've never seen one take an entire 2 hour show.

**Comment by echarlie** [comment]
I'll kick the tires of it a little more; I'd prefer that a 4th person looks over the code and tests the changes before merging it, though.

**Comment by mutantmonkey** [comment]
I'm still waiting for review on this. Don't make me just go ahead and merge it. :P

**Pull request merged**